makerdao/governance-portal-v2
# fetch delegator data in API

### Link to Shortcut ticket:
https://app.shortcut.com/dux-makerdao/story/625/add-list-of-delegators-to-delegate-profile-page

### What does this PR do?
Shows a table of delegators who have delegated to a delegate.

### Steps for testing:
1. go to a delegate page and see the table
2. find a delegate with 0 MKR and don't see the table

### Screenshots (if relevant):
![Screen Shot 2021-11-30 at 6 16 34 PM](https://user-images.githubusercontent.com/13105602/144154905-5082dd5a-bfb7-4ef2-a23f-2235e04e8fa2.png)

### Add a GIF:
![](https://media.giphy.com/media/ZArTeTEhiQkD5uLNdJ/giphy.gif)
